include/ringmesh/mesh/mesh_builder.h

```diff
@@ -768,7 +834,19 @@ namespace RINGMesh
         {
         }
 
+        void update_polygon_attributes()

```

Comment:
I misunderstand this function name. Update for me means update both the size and values of the attributes.

Reply:
Please can you use more precise name